components/AddFormDialog/AddFormDialog.tsx

```diff
@@ -65,22 +67,36 @@ const AddFormDialog = ({
           inPreview: !f.isViewableByAdults && !f.isViewableByChildrens,
         }))
         .concat(
-          gForms.map((f) => ({
-            label: f.text,
-            href: `${f.value}${populateChildForm(
-              person.firstName,
-              person.lastName,
-              person.id,
-              user.email,
-              f.value
-            )}`,
-            system: false,
-            groupRecordable: false,
-            approvable: false,
-            inPreview: false,
-          }))
+          gForms
+            .map((f) => ({
+              label: f.text,
+              href: `${f.value}${populateChildForm(
+                person.firstName,
+                person.lastName,
+                person.id,
+                user.email,
+                f.value
+              )}`,
+              system: false,
+              groupRecordable: false,
+              approvable: false,
+              inPreview: false,
+            }))
+            .concat(
+              isFeatureActive('workflows-pilot') && user.isInWorkflowsPilot
```

Comment:
a suggestion - is it possible to check the current user in the feature flag config?

if so, we could simply use `isFeatureActive` here to keep things clean, and keep the two-part logic of whether the feature is active inside the flag where it belongs:
1. what environment are we on?
2. what group is this user in?

Reply:
Potentially yeah, but my understanding is that we want to show the Workflows link in the dashboard for everyone but the add something new link we only want to show for users in the pilot so I think it needs to link outside of it.